libs/components/datetime/testing/src/modules/datepicker/fuzzy-datepicker-input-harness.ts

```diff
@@ -0,0 +1,34 @@
+import { SkyInputHarness } from '@skyux/core/testing';
+
+/**
+ * Harness to interact with the fuzzy datepicker input harness.
+ */
+export class SkyFuzzyDatepickerInputHarness extends SkyInputHarness {
+  /**
```

Comment:
with this being the same code as SkyDatepickerInputHarness, why not just have this extend that?